Logger/LogFactory.cs

```diff
@@ -1,10 +1,34 @@
-﻿namespace Logger;
-
-public class LogFactory
-{
-    public BaseLogger CreateLogger(string className)
-    {
-        
-        return null;
-    }
-}
+﻿using System;
+using System.Diagnostics;
+
+namespace Logger;
+
+public class LogFactory
+{
+    public string? LogFilePath { get; set; }

```

Comment:
```suggestion
    private string? _FilePath;
```
This should be private and is not an auto property, just a field